TESTS/icb_arpack_cpp.cpp

```diff
@@ -177,8 +177,8 @@ int main() {
   sstats_c();
 
   // arpack without debug
-  real_symmetric_runner<float>();
-  real_symmetric_runner<double>();
+  real_symmetric_runner<float>(1);
+  real_symmetric_runner<double>(1.e-05);
 
   int nopx_c, nbx_c, nrorth_c, nitref_c, nrstrt_c;
```

Comment:
This fails to apply on top of #196, which changes `int` to `a_int` here.

Reply:
#196, #197 and #198 must be tested separately : for each of them, you need to apply all commits of each PR (= 1 branch at my side). Tests must be done separately as they modify the same files: they indeed do conflict, I'll have to rebase them one after the others before they could be merged.

I did 3 PRs as each of them test a different idea/reason. 

Maybe more simple for you (?) to clone my repo (https://github.com/fghoussen/arpack-ng) and, then, switch between branches (issue95 is #196, check is #197 and floatPrecision is #198)

Reply:
I rebased + pushed-force #196 : each PR is one commit now => may be easier to test for you